easybuild/easyconfigs/r/RASPA2/RASPA2-2.0.41-foss-2020b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'RASPA2'
+version = '2.0.41'
+
+homepage = 'https://iraspa.org/raspa/'
+description = """RASPA is a software package for simulating adsorption and diffusion of
+molecules in flexible nanoporous materials."""
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+source_urls = ['https://github.com/iRASPA/RASPA2/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['85d99ad63b83541835f92f09dd41a3c6193a2e4d6ad45baee58ecf3b85713394']
+
+dependencies = [
+    ('OpenBLAS', '0.3.12'),
+    ('FFTW', '3.3.8'),
+    ('ScaLAPACK', '2.1.0'),
+]
+
+preconfigopts = 'rm -rf autom4te.cache && '
+preconfigopts += 'mkdir m4 && '
+preconfigopts += 'aclocal && '
+preconfigopts += 'autoreconf -i && '
+preconfigopts += 'automake --add-missing && '
+preconfigopts += 'autoconf && '
+
+sanity_check_paths = {
+    'files': ['bin/simulate'],
```

Comment:
Can we come up with a simple test command to verify that the binary works, like `simulate --help` maybe?

Reply:
added `simulate -h` (`--help` not supported)